Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor

```diff
@@ -1,6 +1,170 @@
 ﻿@page "/card";
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Card</h2>
+        <p>Cards are a popular and flexible UI component.</p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            A card is just an element with a <Code Content="PENhcmQ+"></Code> applied. You can put any kind of content inside. Make sure you wrap your content in a <Code Content="PENhcmRTZWN0aW9uPg=="></Code> element in order to achieve the traditional card look.
```

Comment:
A card is just an element with a <Card> applied .... its just a copy of the statement based on the fact that class is applied that doesn't make any sense for component... you can even remove this sentence!... element refers to html we are using components not elements.

Reply:
I deleted the first sentence and kept the rest
```suggestion
              You can put any kind of content inside. Make sure you wrap your content in a <Code Content="PENhcmRTZWN0aW9uPg=="></Code> Component in order to achieve the traditional card look.
```